pkg/beacon/relay/node.go

```diff
@@ -43,51 +42,67 @@ func (n *Node) IsInGroup(groupPublicKey []byte) bool {
 	return len(n.groupRegistry.GetGroup(groupPublicKey)) > 0
 }
 
-// JoinGroupIfEligible takes a threshold relay entry value and undergoes the
-// process of joining a group if this node's virtual stakers prove eligible for
-// the group generated by that entry. This is an interactive on-chain process,
-// and JoinGroupIfEligible can block for an extended period of time while it
+// JoinDKGIfEligible takes a seed value and undergoes the process of the
+// distributed key generation if this node's operator proves eligible for
```

Comment:
```suggestion
// distributed key generation if this node's operator proves to be eligible for
```

I think it should be *to be*. It does not come from this PR but we can fix it. Nothing blocking, of course.

Reply:
See https://github.com/keep-network/keep-core/pull/3034/commits/e49262d3683af4ad25eee0a45e031d7f1377770c